### ISSUE TITLE
UnneededCopDisableDirective false positive for RSpec/MultipleExpectations

### ISSUE THREAD

**Issue by quentin-sommer** [rubocop-hq/rubocop#6501]
Lint/UnneededCopDisableDirective cop triggers when the disable statement is needed
--------
## Expected behavior

Rubocop ignores the MultipleExpectations offense

## Actual behavior

Rubocop warns about `Lint/UnneededCopDisableDirective: Unnecessary disabling of RSpec/MultipleExpectations.`

## Steps to reproduce the problem

Using this sample test
```
it 'sets all steps properly' do
  # rubocop:disable RSpec/MultipleExpectations
  expect(1).to have_received(:call).once
  expect(2).to have_received(:call).once.with(user_id: 1234)
  expect(2).to have_received(:call).once.with(user_id: 1234)
  expect(3).to eq([1, 2])
  # rubocop:enable RSpec/MultipleExpectations
end
```
run rubocop. It should output
`Lint/UnneededCopDisableDirective: Unnecessary disabling of RSpec/MultipleExpectations.`

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ [bundle exec] rubocop -V
0.60.0 (using Parser 2.5.3.0, running on ruby 2.5.1 x86_64-darwin17)
```

**Comment by quentin-sommer** [comment]
Looks like I need to disable around the `it` block instead of inside. That made it work
